.github/pages/wait/install/index.css

```
@@ -347,3 +347,157 @@ body.scroll-lock {
 }
+
+/* 入场结束后的「广告」底栏：警戒条 + 虹彩描边，刷新必现、不落存储 */
+@property --cool-notice-angle {
+	syntax: "<angle>";
+	initial-value: 0deg;
+	inherits: false;
+}
+
+.cool-notice {
+	--cool-notice-pad: max(0.75rem, env(safe-area-inset-bottom, 0px));
+	position: fixed;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)